components/NewsItem.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+	<!-- This compenent is used by data from pages api call as well as post api call -->
+	<nuxt-link :to="'/news-and-views/' + slug" class="news-item">
+		<figure class="preview-media">
+			<img :src="picture" class="news-item-img" alt="">
+			<figcaption>
+				<h2>
+					{{ title }}
+				</h2>
+			</figcaption>
+		</figure>
+		<div class="date">
+			<img class="date-icon" alt="">
+			<time>{{ date }}</time>
```

Comment:
Add the proper `datetime` attribute from the API data: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time